modules/administration/pages/ssl-certs-selfsigned.adoc

```diff
@@ -1,9 +1,16 @@
-[[custom-ssl]]
-= Custom SSL Certificates
+[[ssl-certs-selfsigned]]
+= Self-Signed SSL Certificates
 
-You can use custom certificates with {productname} and {productname} Proxy.
+By default, {productname} uses a self-signed certificate.
+In this case, the certificate is created and signed by {productname}.
+This method does not use a third party certificate authority to guarantee that the details of the certificate are correct.
+
+It provides basic checking between the {productname} Server and the clients, but will not protect you against man-in-the-middle attacks.
```

Comment:
This is wrong. Please remove this sentence. If you keep the private key secret, it provide the same protection as a third party certificate.

Reply:
(only line 8)